main.go

```diff
@@ -2,30 +2,56 @@ package main
 
 import (
 	// Import the Discordgo package, and other required packages.
-)
+        "strings"
+	"github.com/bwmarrin/discordgo"
 
-// Init function, called on startup, before the main function.
-func init() {
-	// Setup required variables and other objects here.
-}
+)
 
+//constants are variables that never change
+const (
+token = "token here"
+botPrefix = "!"
+)
 // Main function of the bot, called on startup.
-func main() {
-	// Create a new Discord session using the provided bot token.
-
-	// Register a function as callback for MessageCreate events.
 
-	// Open a websocket connection to Discord and begin listening.
-
-	// Wait until CTRL-C or other term signal is received.
-
-	// Close the websocket gracefully.
+func main() {
+	dg, err := discordgo.New("Bot " + token)
+	if err != nil {
+		panic(err)
+	}
+	u, err := dg.User("@me")
+	if err != nil {
+		panic(err)
+	}
+	botID = u.ID
```

Comment:
botID here has not been initialized. As it is used outside of this function as well, you might have to declare it globally. Please take a look!

Reply:
Im stupid 😀😀